desktop-src/shell/ASSOCF_str.md

```diff
@@ -99,7 +102,17 @@ Specifies that the BaseClass value should be ignored.
 
 **Introduced in Windows 8.1**. Specifies that the ProgID corresponds with a file extension based association. Use together with **ASSOCF\_INIT\_FIXED\_PROGID**.
 
- 
+ <span id="ASSOCF_IS_FULL_URI"></span><span id="assocf_is_full_uri"></span>**ASSOCF\_IS\_FULL\_URI** 
+
+**Introduced in Windows 10 Anniversary Update**. Used to specify that full http/https URI is being passed for target resolution. Only one of ASSOCF_INIT_FIXED_PROGID, ASSOCF_IS_PROTOCOL or ASSOCF_IS_FULL_URI can be specified at a time.
+
+ <span id="ASSOCF_PER_MACHINE_ONLY"></span><span id="assocf_per_machine_only"></span>**ASSOCF\_PER\_MACHINE\_ONLY** 
+
+**Introduced in Windows 10 Anniversary Update**. Enforces per-machine association look-up only and avoid HKCU.
+
+ <span id="ASSOCF_APP_TO_APP"></span><span id="assocf_app_to_app"></span>**ASSOCF\_APP\_TO\_APP** 
+
+**Introduced in Windows 10 April 2018 Update**  For http/https uri associations, this enables selecting the OS default browser, that is a controlled by policy settings, instead of the default browser.
```

Comment:
While I agree that this string matches the header, that's not really what this is for.

This means "Include app-to-app (apps for websites, URI handlers from packaged apps) when resolving."

Reply:
```suggestion
**Introduced in Windows 10 April 2018 Update** Include app-to-app (apps for websites, URI handlers from packaged apps) when resolving.
```